tgui/packages/tgui/interfaces/Fabrication/Types.ts

```diff
@@ -154,3 +154,8 @@ export type FabricatorData = {
     timeLeft: number;
   }[];
 };
+
+// __DEFINES/construction.dm - Sheet Material Amount
+export const SHEET_MATERIAL_AMOUNT = 100;
```

Comment:
Not sure if this is a best way to do it

Reply:
No actually. Because if we do change the values in the dmi files there is a chance we forget to change these.

There is no need to define a constant but just send the defines to the UI as said in 
https://github.com/tgstation/tgstation/pull/76020#discussion_r1227339825

Reply:
No clue how to do it tbh

Reply:
1. pass the define as a variable inside the `ui_static_data()` as shwon below
![Screenshot (258)](https://github.com/tgstation/tgstation/assets/110812394/bf328cdd-1d47-4980-851c-79d0148627ff)
Then access this variable in the UI as use it there

Reply:
This is just an example not specifically for autolathe but for all UI that requires this variable

Reply:
Thanks for help